src/StreamFactoryInterface.php

```diff
@@ -9,9 +9,9 @@
     /**
      * Create a new stream from a resource.
```

Comment:
This comment is also incorrect if a string is allowed. Original implementation: https://github.com/http-interop/http-factory/blob/e4eee1d5c140e699005999671340d6d360f26dff/src/StreamFactoryInterface.php